src/domain/backbone/backbone.repository.spec.ts

```diff
@@ -18,53 +16,27 @@ const transactionApiManager = {
   getTransactionApi: jest.fn(),
 } as unknown as ITransactionApiManager;
 
-const validationErrorFactory = {
-  from: jest.fn(),
-} as unknown as ValidationErrorFactory;
-
-const validationFunction = jest.fn();
-validationFunction.mockImplementation(() => true);
-
-const jsonSchemaService = {
-  addSchema: jest.fn(),
-  compile: jest.fn().mockImplementation(() => validationFunction),
-} as unknown as JsonSchemaService;
+const backboneValidator = {
+  validate: jest.fn().mockResolvedValue(BACKBONE),
```

Comment:
nitpick: from a previous discussion I was under the impression that we'd not mock at this level but inside each test right?